pkg/cmd/codespace/create.go

```diff
@@ -466,12 +466,18 @@ func getRepoSuggestions(ctx context.Context, apiClient apiClient, partialSearch
 }
 
 // buildDisplayName returns display name to be used in the machine survey prompt.
+// prebuildAvailability will be migrated to use enum values: "none", "ready", "in_progress"
```

Comment:
Could you leave a note about how we can know when the migration is complete and we can remove the old values?

Reply:
I think we will finish the migration before Prebuild GA